js/app/modules/card/title.js

```diff
@@ -41,6 +42,22 @@ const Title = new Module.Class({
             'Decorate on highlight', 'Whether to draw a custom decoration when the card is highlighted',
             GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY,
             false),
+
+        /*
+         * Propety: lines
```

Comment:
I thought about that, but I think at least having `title` in the name is good since it refers to the title. This card only has a title, but if we wanted to have a similar property on the other cards, it would be good if it were called the same thing. Otherwise, up to you.